src/bundles/pinning.js

```diff
@@ -102,13 +102,11 @@ const resumePendingPins = (store) => {
 }
 
 const intervalFetchPins = (store) => {
-  setInterval(() => {
-    const pins = [
-      ...store.selectPendingPins(),
-      ...store.selectFailedPins()
-    ].map(serviceCid => ({ cid: serviceCid.split(':')[1] }))
-    store.doFetchRemotePins(pins, true)
-  }, PIN_CHECK_INTERVAL)
+  const pins = [
+    ...store.selectPendingPins(),
+    ...store.selectFailedPins()
+  ].map(serviceCid => ({ cid: serviceCid.split(':')[1] }))
+  store.doFetchRemotePins(pins, true)
 }
```

Comment:
code seems straightforward to me. It makes sense that if this is being called every 30 seconds (`PIN_CHECK_INTERVAL`), that any components dependent upon `doFetchRemotePins` would refresh when this is called. However, if there are no updates, we shouldn't be updating the store and there should be no update.. so this continually updating means the correct fix is probably in `doFetchRemotePins`.